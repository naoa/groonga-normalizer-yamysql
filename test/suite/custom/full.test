register normalizers/yamysql

normalize NormalizerYaMySQLSymbolPartofspeechHtml "**これは<b>テスト</b>データーです。" WITH_TYPES|WITH_CHECKS

