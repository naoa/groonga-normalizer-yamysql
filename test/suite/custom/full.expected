register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLSymbolPartofspeechHtml "**これは<b>テスト</b>データーです。" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "これテストデーター",
    "types": [
      "hiragana",
      "unknown",
      "katakana",
      "katakana",
      "unknown",
      "katakana",
      "katakana",
      "katakana",
      "unknown"
    ],
    "checks": [
      5,
      0,
      0,
      3,
      0,
      0,
      9,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      7,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
